src/test/java/GuiAppTest/TextFieldDouble.java

```diff
@@ -1,11 +1,17 @@
 package GuiAppTest;
 
 import GuiApp.AppTextField;
+import javafx.scene.control.TextField;
 
 public class TextFieldDouble implements AppTextField {
```

Comment:
I think you can call this a `stub`. `TextFieldStub`. I'll give a Zagaku on the different types of test doubles on Tuesday 😃